packages/edit-site/src/store/controls.js

```diff
@@ -15,19 +15,24 @@ import { findTemplate as findTemplateUtil } from '../utils';
  *
  * @return {number} The found template ID.
  */
-export function findTemplate( path ) {
+export function findTemplate( path, templateType, templates ) {
```

Comment:
Please update the function docs with the new params!

Reply:
Done! 🧇